website/events/templates/events/event_cards.html

```diff
@@ -0,0 +1,48 @@
+{% load i18n bleach_tags alert %}
+<section id="events-cards" class="page-section">
+    <div class="container">
+        <h1 class="text-center section-title">{% trans "Upcoming events" %}</h1>
+        <div class="row">
+            {% if not events %}
+                <div class="mt-4 col-12">
+                    {% trans 'There are currently no events planned' as info_text %}
+                    {% alert 'info' info_text dismissible=False %}
+                </div>
+            {% else %}
+                {% for next_event in events %}
+                <div class="col-md-6 col-lg-4">
+                    <a href="{{ next_event.event.get_absolute_url }}">
+                        <div class="card animated fadeIn">
+                            <div class="card-body">
+                                {% if next_event.current_user_registration is not None %}
+                                {% if next_event.current_user_registration %}
+                                    <div class="event-indication" data-toggle="tooltip"
+                                     data-placement="top" title="{% trans "Registered for this event" %}">
+                                        <div class="has-registration"></div>
+                                    </div>
+                                {% else %}
+                                    <div class="event-indication" data-toggle="tooltip"
+                                     data-placement="top" title="{%  trans "Not registered for this event" %}">
+                                        <div class="no-registration"></div>
+                                    </div>
+                                {% endif %}
+                                {% endif %}
```

Comment:
These texts and indicators have changed much more. See the newer implementation on how to set the right text and colour for each event.